repos/system_upgrade/el7toel8/libraries/persistentnetnames.py

```diff
@@ -11,7 +11,7 @@ def physical_interfaces():
     Returns a list of pyudev.Device objects for all physical network interfaces
     """
     enumerator = pyudev.Enumerator(udev_context).match_subsystem('net')
-    return filter(lambda d: not d.device_path.startswith('/devices/virtual/'), enumerator)
+    return (d for d in enumerator if not d.device_path.startswith('/devices/virtual/'))
```

Comment:
List or tuple? "*Returns a list of...*"

Reply:
Yeah, messed up here. Should be a list, thanks!